apps/dashboard/services/graphql/index.ts

```diff
@@ -13,14 +12,14 @@ export const apollo = (token: string) =>
         headers: {
           ["Oauth2-Token"]: token,
         },
-        uri: coreUrl,
+        uri: env.CORE_URL,
         // you can disable result caching here if you want to
         // (this does not work if you are rendering your page with `export const dynamic = "force-static"`)
         fetchOptions: { cache: "no-store" },
       }),
-    })
-  })
+    });
+  });
 
 // TODO: Adds messages only in a dev environment
-loadDevMessages()
-loadErrorMessages()
+loadDevMessages();
+loadErrorMessages();
```

Comment:
not sure that is useful? I think I may have added that for local testing purpose

Reply:
removed